assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -82,6 +86,11 @@ const ACTIONS = {
   confirm: 'confirm' // confirm a submitted extract
 };
 
+// TODO define stargateParams type
+const heyStargate = (stargateParams: any, onAddTaxonomy: Function) => {
+  onAddTaxonomy(stargateParams);
```

Comment:
maybe just call it `addTaxonomy` ?

Reply:
addTaxonomy is already the name of the reducer